universal-application-tool-0.0.1/app/controllers/dev/FileUploadController.java

```diff
@@ -6,44 +6,51 @@
 import com.google.inject.Inject;
 import com.typesafe.config.Config;
 import java.util.Comparator;
-import java.util.Optional;
 import java.util.Set;
 import models.StoredFile;
 import play.Environment;
 import play.mvc.Http.Request;
 import play.mvc.Result;
 import repository.StoredFileRepository;
-import services.aws.SignedS3UploadRequest;
-import services.aws.SimpleStorage;
+import services.cloud.StorageClient;
+import services.cloud.StorageUploadRequest;
 import views.dev.FileUploadView;
 
-/** Controller for interacting with S3 directly in dev mode. */
+/**
+ * Controller for interacting with S3 and blob storage directly in dev mode. The logic for uploading
+ * files to cloud storage has been extracted out into {@link CloudStorageStrategy}
+ */
```

Comment:
issue (non-blocking): I missed that this was originally implemented in `controllers/dev`. We should move it out of this directory and clean up the naming for the class it extends.